src/app/utils/JournalUtils.js

```diff
@@ -6,6 +6,22 @@ import { Modules } from 'coriolis-data/dist';
 import { Modifications } from 'coriolis-data/dist';
 import { getBlueprint, setQualityCB } from './BlueprintFunctions';
 
+/**
+ * Check if an imported module is valid
+ * @param {Object} module the module to check
+ * @param {Object} moduleType the type of module to check
+ * @return {boolean} true if the module is valid
+ */
+function _isValidImportedModule(module, moduleType) {
```

Comment:
Why use the `_...` name? I think this is not standard in the repo. I wouldn't introduce new conventions here.

Reply:
The function declared below it uses the same naming convention, I named it the same in order to stick with the convention in this file.